ReactWindows/ChakraBridge/JsModulesUnbundle.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <stdint.h>
+
+struct JsModulesUnbundleModule sealed
+{
+public:
+	wchar_t* source;
+	wchar_t* sourceUrl;
+	~JsModulesUnbundleModule()
+	{
+		delete[] source;
+		delete[] sourceUrl;
+	}
+};
+
+class JsModulesUnbundle
+{
+public:
+    JsModulesUnbundle(const wchar_t* sourcePath);
+	~JsModulesUnbundle();
+	virtual JsModulesUnbundleModule* GetModule(uint32_t index);
```

Comment:
I would agree except that these unbundles are being used in the context of a native callback, which is expected to return a JsValueRef.  It's not clear how we could effectively transfer a JsErrorCode through the native callback back to the original call to JavaScript from the IJavaScriptExecutor.

---
In reply to: [103549854](https://github.com/ReactWindows/react-native-windows/pull/1017#discussion_r103549854) [](ancestors = 103549854,103549241)